### PR TITLE
volatility 2.0.0

### PR DESCRIPTION
**Pull request by branchv**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

---

For #93940. I enabled some extra packages which were present before ([arch linux](https://archlinux.org/packages/community/any/volatility3/) does as well). I also removed dependencies on `freetype` and `jpeg` since I couldn't tell how they were used, but this may be wrong.

It also has its own [license](https://github.com/volatilityfoundation/volatility3/blob/develop/LICENSE.txt)

### CODE REVIEW

File: Formula/volatility.rb [hunk 1]
```diff
@@ -2,195 +2,68 @@ class Volatility < Formula
...
-    sha256 "86156361c50488b84a3f148056ea716ca587df2f0de1d34750d35c21312725de"
+  resource "future" do
+    url "https://files.pythonhosted.org/packages/45/0b/38b06fd9b92dc2b68d58b75f900e97884c45bedd2ff83203d933cf5851c9/future-0.18.2.tar.gz"
+    sha256 "b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d"
   end
 
-  resource "enum34" do
-    url "https://files.pythonhosted.org/packages/bf/3e/31d502c25302814a7c2f1d3959d2a3b3f78e509002ba91aea64993936876/enum34-1.1.6.tar.gz"
-    sha256 "8ad8c4783bf61ded74527bffb48ed9b54166685e4230386a9ed9b1279e2df5b1"
+  resource "jsonschema" do
```
Comment (SMillerDev) [inline Formula/volatility.rb:37]: There is a formula for this, can't we use that?

File: pypi_formula_mappings.json [hunk 2]
```diff
@@ -486,6 +486,9 @@
   "virtualenv": {
     "exclude_packages": ["six"]
   },
+  "volatility": {
+    "extra_packages": ["capstone", "pycryptodome", "yara-python"]
```
Comment (SMillerDev) [inline pypi_formula_mappings.json:490]: it needs to exclude jsonschema too

Comment (branchvincent) [reply pypi_formula_mappings.json:490]: i don't think so, since in this case `jsonschema` is one of the [extra packages](https://github.com/volatilityfoundation/volatility3/blob/52f325ed54abfdd5d4a35067b7e281fec7881fa4/requirements.txt#L17-L18) we are adding

Comment (carlocab) [reply pypi_formula_mappings.json:490]: But if we use the formula, don't we need it in `exclude_packages`?

Comment (branchvincent) [reply pypi_formula_mappings.json:490]: No, before I added the formula I manually added it in `extra_packages`. So depending on the formula instead just means removing it from `extra_packages`. Note, it is a bit confusing since upstream's [`requirements.txt`](https://github.com/volatilityfoundation/volatility3/blob/52f325ed54abfdd5d4a35067b7e281fec7881fa4/requirements.txt) are not the published dependencies, but rather [`requirements-minimal.txt`](https://github.com/volatilityfoundation/volatility3/blob/52f325ed54abfdd5d4a35067b7e281fec7881fa4/requirements-minimal.txt):

```
$ curl -fsSL https://pypi.org/pypi/volatility3/2.0.0/json | jq -r .info.requires_dist[]
pefile (>=2017.8.1)
```

In other words, running `brew update-python-resources volatility` on this pr results in no changes as expected

**Review by carlocab — APPROVED** [review]
